### DISCUSSION TITLE
Skip Unique Constraint Failure on Seeding?

### DISCUSSION THREAD

**Discussion by darrylmorley** [prisma/prisma#15660] [Q&A]
I expect not, but, is there a quick and easy way to skip a Unique Constraint failure when seeding a DB?

I have a slug field with data that is failing the unique constraint. Is there by any chance a quick way to skip, or mark as undefined, a record that does not meet the unique constraint.

**Comment by austincrim** [comment]
Hey @darrylmorley 👋,

There isn't a way to skip the unique constraint, but depending on your use case you might be able to use [`upsert`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#upsert) or [$queryRaw](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw) to accomplish what you need.

**Reply by vladi-stevanovic** [reply]
Hello @darrylmorley , did you perhaps have a chance to try the workaround Austin suggested? Let us know if you still need help here!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
